### PR TITLE
Fix missing signal for allocated size changes

### LINKED ISSUE #2863: PropertiesChanged signal missing for property "AllocatedSize" in pool when add-data is done
If you start with a pool like:

```
# dbus-send --system --dest=org.storage.stratis3 --print-reply /org/storage/stratis3/4 org.freedesktop.DBus.Properties.GetAll  string:org.storage.stratis3.pool.r0
method return time=1638461124.056256 sender=:1.18 -> destination=:1.167 serial=37 reply_serial=2
   array [
      dict entry(
         string "AllocatedSize"
         variant             string "2147483648"
      )
      dict entry(
         string "AvailableActions"
         variant             string "fully_operational"
      )
      dict entry(
         string "ClevisInfo"
         variant             struct {
               boolean true
               struct {
                  boolean false
                  struct {
                     string ""
                     string ""
                  }
               }
            }
      )
      dict entry(
         string "Encrypted"
         variant             boolean false
      )
      dict entry(
         string "HasCache"
         variant             boolean false
      )
      dict entry(
         string "KeyDescription"
         variant             struct {
               boolean true
               struct {
                  boolean false
                  string ""
               }
            }
      )
      dict entry(
         string "Name"
         variant             string "some_pool"
      )
      dict entry(
         string "TotalPhysicalSize"
         variant             string "2147483648"
      )
      dict entry(
         string "TotalPhysicalUsed"
         variant             struct {
               boolean true
               string "43655168"
            }
      )
      dict entry(
         string "Uuid"
         variant             string "c7bd124af0b4471695ce25fc5f3a3f42"
      )
   ]
```
And you do:

```bash
# stratis pool add-data some_pool /dev/sdd
```

You get the following signals.

```
# dbus-monitor --system "type='signal',sender='org.storage.stratis3'"
signal time=1638461224.030193 sender=org.freedesktop.DBus -> destination=:1.170 serial=4294967295 path=/org/freedesktop/DBus; interface=org.freedesktop.DBus; member=NameAcquired
   string ":1.170"
signal time=1638461224.030251 sender=org.freedesktop.DBus -> destination=:1.170 serial=4294967295 path=/org/freedesktop/DBus; interface=org.freedesktop.DBus; member=NameLost
   string ":1.170"
signal time=1638461252.870471 sender=:1.18 -> destination=(null destination) serial=40 path=/org/storage/stratis3/4; interface=org.freedesktop.DBus.Properties; member=PropertiesChanged
   string "org.storage.stratis3.pool.r0"
   array [
      dict entry(
         string "TotalPhysicalSize"
         variant             string "2201169690624"
      )
   ]
   array [
   ]
signal time=1638461252.870764 sender=:1.18 -> destination=(null destination) serial=42 path=/org/storage/stratis3; interface=org.freedesktop.DBus.ObjectManager; member=InterfacesAdded
   object path "/org/storage/stratis3/7"
   array [
      dict entry(
         string "org.storage.stratis3.blockdev.r0"
         array [
            dict entry(
               string "Uuid"
               variant                   string "2d81dbc7f1bb4e6d9c16f6cd3a0f489c"
            )
            dict entry(
               string "HardwareInfo"
               variant                   struct {
                     boolean false
                     string ""
                  }
            )
            dict entry(
               string "PhysicalPath"
               variant                   string "/dev/sdd"
            )
            dict entry(
               string "Tier"
               variant                   uint16 0
            )
            dict entry(
               string "TotalPhysicalSize"
               variant                   string "2199022206976"
            )
            dict entry(
               string "Devnode"
               variant                   string "/dev/sdd"
            )
            dict entry(
               string "UserInfo"
               variant                   struct {
                     boolean false
                     string ""
                  }
            )
            dict entry(
               string "Pool"
               variant                   object path "/org/storage/stratis3/4"
            )
            dict entry(
               string "InitializationTime"
               variant                   uint64 1638461252
            )
         ]
      )
   ]
```

If you retrieve the pool properties again:

```
# dbus-send --system --dest=org.storage.stratis3 --print-reply /org/storage/stratis3/4 org.freedesktop.DBus.Properties.GetAll  string:org.storage.stratis3.pool.r0
method return time=1638461341.516469 sender=:1.18 -> destination=:1.172 serial=43 reply_serial=2
   array [
      dict entry(
         string "AllocatedSize"
         variant             string "2201169690624"
      )
      dict entry(
         string "AvailableActions"
         variant             string "fully_operational"
      )
      dict entry(
         string "ClevisInfo"
         variant             struct {
               boolean true
               struct {
                  boolean false
                  struct {
                     string ""
                     string ""
                  }
               }
            }
      )
      dict entry(
         string "Encrypted"
         variant             boolean false
      )
      dict entry(
         string "HasCache"
         variant             boolean false
      )
      dict entry(
         string "KeyDescription"
         variant             struct {
               boolean true
               struct {
                  boolean false
                  string ""
               }
            }
      )
      dict entry(
         string "Name"
         variant             string "some_pool"
      )
      dict entry(
         string "TotalPhysicalSize"
         variant             string "2201169690624"
      )
      dict entry(
         string "TotalPhysicalUsed"
         variant             struct {
               boolean true
               string "2244644864"
            }
      )
      dict entry(
         string "Uuid"
         variant             string "c7bd124af0b4471695ce25fc5f3a3f42"
      )
   ]
```

You will discover that the "AllocatedSize" property change did not get published in the `PropertiesChanged` signal.

I'm using a utility I wrote for lvmdbusd to discover these [inconsistencies](https://github.com/tasleson/misc-tools/blob/master/oversight/oversight.py).  At the moment I simply start the utility and exercise stratisd with the command line and/or scripts.

When testing lvmdbusd I make a slight adjustment to the tear down method to induce a delay, so that this utility can verify state.  Otherwise it's difficult to verify correctness when things are constantly changing.

### PR DESCRIPTION
**Pull request by jbaublitz**
Closes #2863 

Hi @tasleson feel free to verify this one as well! I've tested on my end.

### CODE REVIEW

**Comment by tasleson** [comment]
> Closes #2863
> 
> Hi @tasleson feel free to verify this one as well! I've tested on my end.

I believe this is corrected.  However, I may be doing something wrong as the `TotalPhysicalUsed` is now incorrect again during my testing, but I had to resolve a conflict to apply this fix.  Would it be possible for you to create a branch that has all three fixes applied for me to test to rule out me doing something incorrect?

Thanks

File: src/dbus_api/pool/shared.rs [hunk 1]
```diff
@@ -108,20 +108,27 @@ where
...
                 dbus_context,
                 pool_path.get_name()
-            );
-            dbus_context
-                .push_pool_size_change(pool_path.get_name(), pool.total_physical_size().bytes());
-            res
+            )
+        }
+        BlockDevOp::AddData => {
+            handle_action!(pool
```
Comment (mulkieran) [inline src/dbus_api/pool/shared.rs:120]: I think you need to stick with the ```handle_action!``` variant that takes 3 args.

**Review by mulkieran — CHANGES_REQUESTED** [review]
One request, otherwise, looks good to me.

**Comment by jbaublitz** [comment]
> > Closes #2863
> > Hi @tasleson feel free to verify this one as well! I've tested on my end.
> 
> I believe this is corrected. However, I may be doing something wrong as the `TotalPhysicalUsed` is now incorrect again during my testing, but I had to resolve a conflict to apply this fix. Would it be possible for you to create a branch that has all three fixes applied for me to test to rule out me doing something incorrect?
> 
> Thanks

I am about to rebase this one and the other fix is merged so this branch should resolve those issues. I am still unsure of what the best fix is for #2864 in terms of the MDV, but I am considering putting up a PR that resolves #2824 and the MDV issue. I think it would make sense to resolve a lot of the retry errors all at once. I can rebase one of the branches onto the other so you can test it.

**Review by mulkieran — APPROVED** [review]

**Comment by tasleson** [comment]
@jbaublitz With these two commits cherry-picked on master I'm getting a failure with `TotalPhysicalUsed`, but the `AllocatedSize` is correct.

This is the utility output below (I added dumping object state on object add signal), starting from blank state and issuing

```bash
./stratis pool create the_pool /dev/sdb /dev/sdc
# ~92.5 second delay
./stratis pool add-data the_pool /dev/sdd

```

```
# ./oversight.py org.storage.stratis3 /org/storage/stratis3
[1638900828.220198][17.526301]: Object add: /org/storage/stratis3/0
[1638900828.220356][0.000157]:  
[1638900828.220394][0.000038]: object /org/storage/stratis3/0
[1638900828.220439][0.000046]: Interface org.storage.stratis3.pool.r0
[1638900828.220494][0.000055]: 	AllocatedSize: 2147483648
[1638900828.220526][0.000031]: 	AvailableActions: fully_operational
[1638900828.220601][0.000076]: 	ClevisInfo: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.Struct((dbus.String(''), dbus.String('')), signature=None)), signature=None)), signature=None, variant_level=1)
[1638900828.220641][0.000040]: 	Encrypted: 0
[1638900828.220669][0.000028]: 	HasCache: 0
[1638900828.220715][0.000046]: 	KeyDescription: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None)), signature=None, variant_level=1)
[1638900828.220782][0.000067]: 	Name: the_pool
[1638900828.220810][0.000029]: 	TotalPhysicalSize: 2147483648
[1638900828.220925][0.000115]: 	TotalPhysicalUsed: dbus.Struct((dbus.Boolean(True), dbus.String('43655168')), signature=None, variant_level=1)
[1638900828.220963][0.000037]: 	Uuid: 7441c6dbe7c9406dba1a703061251a13
[1638900828.221305][0.000342]: Object add: /org/storage/stratis3/1
[1638900828.221348][0.000043]:  
[1638900828.221378][0.000030]: object /org/storage/stratis3/1
[1638900828.221420][0.000042]: Interface org.storage.stratis3.blockdev.r0
[1638900828.221472][0.000053]: 	Devnode: /dev/sdb
[1638900828.221523][0.000051]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900828.221558][0.000035]: 	InitializationTime: 1638900828
[1638900828.221589][0.000031]: 	PhysicalPath: /dev/sdb
[1638900828.221618][0.000029]: 	Pool: /org/storage/stratis3/0
[1638900828.221649][0.000031]: 	Tier: 0
[1638900828.221679][0.000030]: 	TotalPhysicalSize: 1073741824
[1638900828.221813][0.000134]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900828.221856][0.000043]: 	Uuid: 2290c6ace5534d8fb1c8fb03bacf9600
[1638900828.222599][0.000742]: Object add: /org/storage/stratis3/2
[1638900828.222707][0.000108]:  
[1638900828.222775][0.000067]: object /org/storage/stratis3/2
[1638900828.222819][0.000045]: Interface org.storage.stratis3.blockdev.r0
[1638900828.222874][0.000055]: 	Devnode: /dev/sdc
[1638900828.223006][0.000131]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900828.223045][0.000040]: 	InitializationTime: 1638900828
[1638900828.223452][0.000407]: 	PhysicalPath: /dev/sdc
[1638900828.223507][0.000055]: 	Pool: /org/storage/stratis3/0
[1638900828.223540][0.000033]: 	Tier: 0
[1638900828.223569][0.000029]: 	TotalPhysicalSize: 1073741824
[1638900828.224033][0.000464]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900828.224401][0.000368]: 	Uuid: a2b9093e461a41ef8452927267f4879d
[1638900831.223982][2.999581]: Validating objects entry
[1638900831.331595][0.107613]: Validating objects exit 0.107606
[1638900923.911175][92.579581]: /org/storage/stratis3/0[org.storage.stratis3.pool.r0][TotalPhysicalSize] = 2201169690624
[1638900923.911308][0.000132]: /org/storage/stratis3/0[org.storage.stratis3.pool.r0][AllocatedSize] = 2201169690624
[1638900923.911431][0.000123]: Object add: /org/storage/stratis3/3
[1638900923.911488][0.000057]:  
[1638900923.911512][0.000024]: object /org/storage/stratis3/3
[1638900923.911531][0.000020]: Interface org.storage.stratis3.blockdev.r0
[1638900923.911552][0.000020]: 	Devnode: /dev/sdd
[1638900923.911572][0.000021]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900923.911589][0.000017]: 	InitializationTime: 1638900923
[1638900923.911596][0.000007]: 	PhysicalPath: /dev/sdd
[1638900923.911620][0.000024]: 	Pool: /org/storage/stratis3/0
[1638900923.911637][0.000017]: 	Tier: 0
[1638900923.911693][0.000056]: 	TotalPhysicalSize: 2199022206976
[1638900923.911716][0.000023]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900923.911787][0.000071]: 	Uuid: e40136a5a5504bca8f339397eba0b352
[1638900927.349978][3.438191]: Validating objects entry
[1638900927.456043][0.106064]: Property (TotalPhysicalUsed) mismatch objectmgr dbus.Struct((dbus.Boolean(True), dbus.String('2244644864')), signature=None, variant_level=1) !=  signal value: dbus.Struct((dbus.Boolean(True), dbus.String('43655168')), signature=None) object: /org/storage/stratis3/0
[1638900927.456151][0.000108]: Signal db
[1638900927.456178][0.000027]:  
[1638900927.456198][0.000020]: object /org/storage/stratis3/0
[1638900927.456228][0.000030]: Interface org.storage.stratis3.pool.r0
[1638900927.456264][0.000035]: 	AllocatedSize: 2201169690624
[1638900927.456283][0.000019]: 	AvailableActions: fully_operational
[1638900927.456388][0.000105]: 	ClevisInfo: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.Struct((dbus.String(''), dbus.String('')), signature=None)), signature=None)), signature=None, variant_level=1)
[1638900927.456453][0.000065]: 	Encrypted: 0
[1638900927.456504][0.000051]: 	HasCache: 0
[1638900927.456567][0.000063]: 	KeyDescription: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None)), signature=None, variant_level=1)
[1638900927.456656][0.000089]: 	Name: the_pool
[1638900927.456713][0.000056]: 	TotalPhysicalSize: 2201169690624
[1638900927.456774][0.000061]: 	TotalPhysicalUsed: dbus.Struct((dbus.Boolean(True), dbus.String('43655168')), signature=None, variant_level=1)
[1638900927.456995][0.000221]: 	Uuid: 7441c6dbe7c9406dba1a703061251a13
[1638900927.457089][0.000094]: GetManagedObjects
[1638900927.457157][0.000069]:  
[1638900927.457206][0.000049]: object /org/storage/stratis3/0
[1638900927.457253][0.000047]: Interface org.storage.stratis3.pool.r0
[1638900927.457307][0.000054]: 	AllocatedSize: 2201169690624
[1638900927.457360][0.000053]: 	AvailableActions: fully_operational
[1638900927.457435][0.000076]: 	ClevisInfo: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.Struct((dbus.String(''), dbus.String('')), signature=None)), signature=None)), signature=None, variant_level=1)
[1638900927.457515][0.000080]: 	Encrypted: 0
[1638900927.457694][0.000180]: 	HasCache: 0
[1638900927.457987][0.000293]: 	KeyDescription: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None)), signature=None, variant_level=1)
[1638900927.458059][0.000072]: 	Name: the_pool
[1638900927.458111][0.000053]: 	TotalPhysicalSize: 2201169690624
[1638900927.458210][0.000098]: 	TotalPhysicalUsed: dbus.Struct((dbus.Boolean(True), dbus.String('2244644864')), signature=None, variant_level=1)
[1638900927.458347][0.000137]: 	Uuid: 7441c6dbe7c9406dba1a703061251a13
[1638900927.458573][0.000226]: Validating objects exiting ON ERROR! duration = 0.108590
[1638900927.995612][0.537039]: **** Dumping signal db
[1638900927.995671][0.000058]:  
[1638900927.995685][0.000014]: object /org/storage/stratis3/0
[1638900927.995703][0.000018]: Interface org.storage.stratis3.pool.r0
[1638900927.995725][0.000022]: 	AllocatedSize: 2201169690624
[1638900927.995736][0.000010]: 	AvailableActions: fully_operational
[1638900927.995864][0.000128]: 	ClevisInfo: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.Struct((dbus.String(''), dbus.String('')), signature=None)), signature=None)), signature=None, variant_level=1)
[1638900927.995888][0.000023]: 	Encrypted: 0
[1638900927.995913][0.000026]: 	HasCache: 0
[1638900927.995927][0.000014]: 	KeyDescription: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None)), signature=None, variant_level=1)
[1638900927.995936][0.000009]: 	Name: the_pool
[1638900927.995943][0.000007]: 	TotalPhysicalSize: 2201169690624
[1638900927.995953][0.000011]: 	TotalPhysicalUsed: dbus.Struct((dbus.Boolean(True), dbus.String('2244644864')), signature=None, variant_level=1)
[1638900927.995981][0.000027]: 	Uuid: 7441c6dbe7c9406dba1a703061251a13
[1638900927.996005][0.000024]:  
[1638900927.996020][0.000015]: object /org/storage/stratis3/1
[1638900927.996039][0.000019]: Interface org.storage.stratis3.blockdev.r0
[1638900927.996069][0.000031]: 	Devnode: /dev/sdb
[1638900927.996096][0.000026]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900927.996119][0.000024]: 	InitializationTime: 1638900828
[1638900927.996140][0.000021]: 	PhysicalPath: /dev/sdb
[1638900927.996161][0.000021]: 	Pool: /org/storage/stratis3/0
[1638900927.996182][0.000021]: 	Tier: 0
[1638900927.996209][0.000027]: 	TotalPhysicalSize: 1073741824
[1638900927.996248][0.000039]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900927.996313][0.000065]: 	Uuid: 2290c6ace5534d8fb1c8fb03bacf9600
[1638900927.996339][0.000026]:  
[1638900927.996354][0.000015]: object /org/storage/stratis3/2
[1638900927.996374][0.000019]: Interface org.storage.stratis3.blockdev.r0
[1638900927.996407][0.000033]: 	Devnode: /dev/sdc
[1638900927.996448][0.000041]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900927.996473][0.000025]: 	InitializationTime: 1638900828
[1638900927.996494][0.000021]: 	PhysicalPath: /dev/sdc
[1638900927.996515][0.000021]: 	Pool: /org/storage/stratis3/0
[1638900927.996535][0.000020]: 	Tier: 0
[1638900927.996563][0.000028]: 	TotalPhysicalSize: 1073741824
[1638900927.996633][0.000070]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900927.996680][0.000047]: 	Uuid: a2b9093e461a41ef8452927267f4879d
[1638900927.996705][0.000025]:  
[1638900927.996792][0.000087]: object /org/storage/stratis3/3
[1638900927.996820][0.000028]: Interface org.storage.stratis3.blockdev.r0
[1638900927.996844][0.000024]: 	Devnode: /dev/sdd
[1638900927.996872][0.000028]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900927.996911][0.000039]: 	InitializationTime: 1638900923
[1638900927.996943][0.000032]: 	PhysicalPath: /dev/sdd
[1638900927.996965][0.000022]: 	Pool: /org/storage/stratis3/0
[1638900927.996985][0.000021]: 	Tier: 0
[1638900927.997005][0.000020]: 	TotalPhysicalSize: 2199022206976
[1638900927.997031][0.000026]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900927.997053][0.000022]: 	Uuid: e40136a5a5504bca8f339397eba0b352
[1638900927.997087][0.000033]: **** Dumping GetManagedObjects

[1638900928.099972][0.102885]:  
[1638900928.100129][0.000157]: object /org/storage/stratis3/2
[1638900928.100191][0.000062]: Interface org.storage.stratis3.blockdev.r0
[1638900928.100253][0.000062]: 	Devnode: /dev/sdc
[1638900928.100335][0.000082]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900928.100417][0.000082]: 	InitializationTime: 1638900828
[1638900928.100480][0.000063]: 	PhysicalPath: /dev/sdc
[1638900928.100532][0.000053]: 	Pool: /org/storage/stratis3/0
[1638900928.100583][0.000051]: 	Tier: 0
[1638900928.100671][0.000087]: 	TotalPhysicalSize: 1073741824
[1638900928.100738][0.000067]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900928.101016][0.000278]: 	Uuid: a2b9093e461a41ef8452927267f4879d
[1638900928.101104][0.000089]:  
[1638900928.101167][0.000063]: object /org/storage/stratis3/3
[1638900928.101243][0.000076]: Interface org.storage.stratis3.blockdev.r0
[1638900928.101303][0.000059]: 	Devnode: /dev/sdd
[1638900928.101733][0.000431]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900928.101976][0.000242]: 	InitializationTime: 1638900923
[1638900928.102089][0.000113]: 	PhysicalPath: /dev/sdd
[1638900928.102142][0.000053]: 	Pool: /org/storage/stratis3/0
[1638900928.102243][0.000102]: 	Tier: 0
[1638900928.102329][0.000085]: 	TotalPhysicalSize: 2199022206976
[1638900928.102401][0.000073]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900928.102457][0.000055]: 	Uuid: e40136a5a5504bca8f339397eba0b352
[1638900928.102510][0.000053]:  
[1638900928.102547][0.000037]: object /org/storage/stratis3/0
[1638900928.102593][0.000046]: Interface org.storage.stratis3.pool.r0
[1638900928.102720][0.000127]: 	AllocatedSize: 2201169690624
[1638900928.102953][0.000233]: 	AvailableActions: fully_operational
[1638900928.103046][0.000093]: 	ClevisInfo: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.Struct((dbus.String(''), dbus.String('')), signature=None)), signature=None)), signature=None, variant_level=1)
[1638900928.103118][0.000072]: 	Encrypted: 0
[1638900928.103204][0.000086]: 	HasCache: 0
[1638900928.103320][0.000116]: 	KeyDescription: dbus.Struct((dbus.Boolean(True), dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None)), signature=None, variant_level=1)
[1638900928.103415][0.000096]: 	Name: the_pool
[1638900928.103501][0.000086]: 	TotalPhysicalSize: 2201169690624
[1638900928.103687][0.000186]: 	TotalPhysicalUsed: dbus.Struct((dbus.Boolean(True), dbus.String('2244644864')), signature=None, variant_level=1)
[1638900928.104000][0.000313]: 	Uuid: 7441c6dbe7c9406dba1a703061251a13
[1638900928.104111][0.000111]:  
[1638900928.104181][0.000070]: object /org/storage/stratis3/1
[1638900928.104245][0.000064]: Interface org.storage.stratis3.blockdev.r0
[1638900928.104299][0.000055]: 	Devnode: /dev/sdb
[1638900928.104373][0.000073]: 	HardwareInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900928.104433][0.000060]: 	InitializationTime: 1638900828
[1638900928.104486][0.000054]: 	PhysicalPath: /dev/sdb
[1638900928.104567][0.000081]: 	Pool: /org/storage/stratis3/0
[1638900928.104903][0.000335]: 	Tier: 0
[1638900928.104985][0.000082]: 	TotalPhysicalSize: 1073741824
[1638900928.105083][0.000098]: 	UserInfo: dbus.Struct((dbus.Boolean(False), dbus.String('')), signature=None, variant_level=1)
[1638900928.105162][0.000079]: 	Uuid: 2290c6ace5534d8fb1c8fb03bacf9600
```

**Comment by jbaublitz** [comment]
@tasleson @mulkieran I'm going to rebase and resolve conflicts but I've verified that this PR resolves the problem.

**Comment by mulkieran** [comment]
Thanks, it is in.

**Comment by jbaublitz** [comment]
@tasleson @mulkieran Can I get one last review on this now that the conflicts are resolved?

**Review by mulkieran — APPROVED** [review]

**Comment by mulkieran** [comment]
@jbaublitz Plz rebase, thx.

**Comment by jbaublitz** [comment]
@mulkieran Rebased!

**Comment by mulkieran** [comment]
@jbaublitz I'm feeling concerned that both the ```checks_with_udev``` tests seem to have been stalled on ```test engine::strat_engine::pool::tests::loop_test_add_datadevs```. What do you think?

**Comment by jbaublitz** [comment]
@mulkieran The thin pool rework PR tests stalled at a different point. I'm wondering if there's some infrastructure issues with Github Actions right now.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**